swagger.yaml

```diff
@@ -3754,6 +3754,49 @@ paths:
           schema:
             $ref: '#/definitions/error'
 
+  /v2/infra-envs/{infra_env_id}/downloads/minimal-initrd:
+    get:
+      tags:
+        - installer
+      security:
+        - userAuth: [admin, read-only-admin, user]
```

Comment:
This should also have `urlAuth: []` like the files download for when we run with the operator.

Reply:
Added.